### ISSUE TITLE
How long does it take to train

### ISSUE THREAD

**Issue by djFatNerd** [hubert0527/infinityGAN#17]
Dear Authors:

Thank you for your great work.

I'm trying to train InfinityGAN with my own dataset which consistes of ~80,000 256 x256 images. I am using a batch size of 32 training on 4 x 3090 GPUs, and I have trained for about 150,000 iterations.

However, the FIDs are still high and I am not getting any visible results.  I have attached some sampled results and training stats below, may I ask you if these look normal to you? Does these indicate I might have made some mistakes somewhere?   Thank you so much for you help!

<img width="181" alt="0a4a0c52b8dd6b86cbb72375d8b8fae" src="https://github.com/hubert0527/infinityGAN/assets/44420977/24320bc9-4a7a-4a2d-a3c8-6a4b347811fc">
<img width="887" alt="6f8200aa3bacaa3cd32999898a85ec0" src="https://github.com/hubert0527/infinityGAN/assets/44420977/33d687b3-fcf8-4765-a214-2f533030741f">
<img width="877" alt="202dc9ba9156dbf0acbf72267161bc8" src="https://github.com/hubert0527/infinityGAN/assets/44420977/e7551178-c893-44d5-971c-cb5c161f02b4">
<img width="867" alt="d0fab649b583a36ea2425a0f0489b46" src="https://github.com/hubert0527/infinityGAN/assets/44420977/d581c3d6-ad30-470e-801e-8f068cf1e66a">
<img width="205" alt="d97d2e2996329782437c3be0e5ec184" src="https://github.com/hubert0527/infinityGAN/assets/44420977/12bcab43-14aa-4bbc-9a91-b89b162e9db0">
<img width="857" alt="e35bb62f5dfd9fb2535aeee2485c916" src="https://github.com/hubert0527/infinityGAN/assets/44420977/676e9cff-4421-484b-a609-1e9e047aaa31">
<img width="221" alt="fa1d500b20440b3ec8eed8bcb675fbd" src="https://github.com/hubert0527/infinityGAN/assets/44420977/b0a52958-3634-46dc-9175-0191c810fa2e">

**Comment by hubert0527** [comment]
This is abnormal to me. The generator should start generating some initial structured pixels around 30k iterations.
I would try to increase the gradient penalties and reduce the diversity loss.
There could be some other issues with your configurations, as "structure_diversity_D_score" has a very weird distribution.

**Comment by djFatNerd** [comment]
Hi, thank you for your reply. We used the exact same configuration as "InfinityGAN-IOP.yaml"

We found the training process is highly sensitive to learning rate and we had to adjust the learning rate frequently during training.

We are now able to get some results at ~200,000 epochs and reaching an FID of ~80.

<img width="566" alt="1700898553127" src="https://github.com/hubert0527/infinityGAN/assets/44420977/77cbf179-03f2-48b9-af4e-a4d357afc4bb">
